sandbox-tools/merge-repos/src/tests/karma.ts

```diff
@@ -160,12 +160,19 @@ export async function createPackageKarmaConfig(git: SimpleGit, basePath: string,
                 await git.add(webpackCfg);
 
                 addWebpack = true;
+            } else {
+                log(` -- ${path.join(dest, "test/worker/index-webpack.worker.ts")} -- missing`);
```

Comment:
This was additional debug messages which identified the issues with the auto adding of webworker tests.